clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -533,7 +539,9 @@ void ProgramInfo::addVariable(clang::DeclaratorDecl *D,
     // the parameters.
     for (unsigned i = 0; i < FD->getNumParams(); i++) {
       ParmVarDecl *PVD = FD->getParamDecl(i);
+      const Type *Ty = PVD->getType().getTypePtr();
       ConstraintVariable *PV = F->getParamVar(i);
+      checkTypedef(Ty, *AstContext, PVD, dyn_cast<PVConstraint>(PV));
```

Comment:
I notice that most places in this funciton we check if the thing being added is a pointer type, but we don't do that here. I wonder why?